app/api/helpers/db.py

```diff
@@ -142,3 +142,12 @@ def get_new_slug(model, name):
         return slug
     else:
         return '{}-{}'.format(slug, uuid.uuid4().hex)
+
+def get_new_identifier(model):
+    """ param model: Specify model from db"""
+    identifier = str(uuid.uuid4())
+    count = get_count(model.query.filter_by(identifier=identifier))
+    if count == 0:
+        return identifier
+    else:
+        return get_new_identifier()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'model' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5550207)